packages/adaptive_scaffold/example/test/adaptive_layout_demo_test.dart

```diff
@@ -2,11 +2,266 @@
 // Use of this source code is governed by a BSD-style license that can be
 // found in the LICENSE file.
 
+import 'package:adaptive_scaffold/src/adaptive_layout.dart';
+import 'package:adaptive_scaffold/src/slot_layout.dart';
+import 'package:adaptive_scaffold/src/breakpoints.dart';
```

Comment:
You can remove these as they are imported by adaptive_scaffold.dart in the next line.